pkg/server/sandbox_run.go

```diff
@@ -639,3 +641,15 @@ func (c *criService) getSandboxRuntime(config *runtime.PodSandboxConfig, runtime
 	}
 	return handler, nil
 }
+
+func logDebugCNIResult(sandboxID string, result *cni.CNIResult) {
+	if logrus.GetLevel() != logrus.DebugLevel {
+		return
+	}
+	cniResult, err := json.Marshal(result)
+	if err != nil {
+		logrus.WithError(err).Errorf("Failed to marshal CNI result for sandbox %q: %v", sandboxID, err)
+		return
+	}
+	logrus.WithField("cni-result", string(cniResult)).Debugf("cni result for sandbox %q", sandboxID)
```

Comment:
This would be the only use of field descriptive log output in the cri code. No need to say "cni-result" and "cni result for." So I'd pick one, with field if you think that is needed / have a reason for the field descriptive logging and remove the "cni result for" language, or just use logsrus.Debugf

Reply:
Sure this is quite redundant, I'll follow the codestyle of the project and don't use `WithField` 👌